plugins/test/unit/plugins/importers/yum/repomd/test_metadata.py

```diff
@@ -90,6 +93,35 @@ def test_add_repo_data_filters_location_tag(self, mock_filelists, mock_other, mo
         mock_change_location_tag.assert_called_once_with(raw_xml, model.relative_path)
         self.assertEquals('baz', model.metadata['repodata']['primary'])
 
+    def test_get_metadata_file_bz(self):
+        working_dir = tempfile.mkdtemp()
+        # create the test file
+        source_file = os.path.join(working_dir, 'foo.bz2')
+        with bz2.BZ2File(source_file, 'w') as compressed_file_handle:
```

Comment:
I don't have a python 2.6 install handy, but this might only work on 2.7 and newer.

https://docs.python.org/2/library/bz2.html